packages/docusaurus-theme-redoc/src/theme/Redoc/styles.css

```diff
@@ -71,18 +71,14 @@ html:not([data-theme="dark"]) .redocusaurus div[id^="react-tabs"] > div:nth-chil
 
 /* ------ Schema Styling Overrides ------- */
 
-.redocusaurus table th, table td {
+.redocusaurus table th, .redocusaurus table td {
```

Comment:
This will prevent the table styles from being applied onto other tables that are not in redoc